base/lthooks.dtx

```diff
@@ -6163,6 +6163,7 @@
 %
 %
 % \begin{macro}{\hook_show:n,\hook_log:n}
+% \changes{v1.1l}{2025/06/27}{Support \cs{showstream} (gh/1062)}
```

Comment:
```suggestion
% \changes{v1.1m}{2025/06/27}{Support \cs{showstream} (gh/1062)}
```

Reply:
This is marked resolved while not applied?

This PR bumps the version of `base/lthooks.dtx` from 1.1l to 1.1m, so the newly added `\changes` entry should use 1.1m too.

Reply:
Should now be sorted